src/Microsoft.ML.Transforms/CountFeatureSelection.cs

```diff
@@ -18,7 +18,47 @@
 
 namespace Microsoft.ML.Transforms
 {
-    /// <include file='doc.xml' path='doc/members/member[@name="CountFeatureSelection"]' />
+    /// <summary>
+    ///  Selects the slots for which the count of non-default values is greater than or equal to a threshold.
```

Comment:
>default values [](start = 54, length = 14)

create new issue for later: we need to specify for each input type what's the default value. for e.g. it's not clear if for text, default is null, empty string, or whitespaces. same goes with key type.

Reply:
Created: [https://github.com/dotnet/machinelearning/issues/3443](https://github.com/dotnet/machinelearning/issues/3443)

---
In reply to: [277027260](https://github.com/dotnet/machinelearning/pull/3424#discussion_r277027260) [](ancestors = 277027260)